docs/framework/wcf/feature-details/elevation-of-privilege.md

```diff
@@ -32,7 +32,7 @@ ms.assetid: 146e1c66-2a76-4ed3-98a5-fd77851a06d9
  If you want to have deterministic behavior when using Windows authentication together with impersonation you need to explicitly set the Windows credential or you need to establish a security context with the service. To do this, use a message security session or a transport security session. For example, the net.tcp transport can provide a transport security session. Additionally, you must use only a synchronous version of client operations when calling the service. If you establish a message security context, you should not keep the connection to the service open longer than the configured session renewal period, because the identity can also change during the session renewal process.  
   
 ### Credentials Capture  
- The following applies to [!INCLUDE[netfx35_long](../../../../includes/netfx35-long-md.md)], and subsequent versions.  
+ The following applies to .NET Framework version 3.5, and subsequent versions.  
```

Comment:
```suggestion
 The following applies to .NET Framework 3.5, and subsequent versions.  
```

Shall I remove other "version" strings?

Reply:
Yep, if it's related to the changes you're making